roles/pulp_common/tasks/configure.yml

```
@@ -47,7 +47,7 @@
     - include_tasks: prepare_signing_service.yml
       when:
         - developer_user is defined or developer_user_home is defined or __galaxy_build_signing_scripts is defined
-        - galaxy_create_default_collection_signing_service
+        - galaxy_create_default_collection_signing_service is defined and galaxy_create_default_collection_signing_service is true
```

FYI: You can also do:
```suggestion
        - galaxy_create_default_collection_signing_service | default(false)
```
Or put it in the defaults file.